resim/metrics/proto/metrics.proto

```diff
@@ -461,3 +463,7 @@ message PlotlyMetricValues {
 message ImageMetricValues {
     MetricsDataId image_data_id = 1;
 }
+
+message TextMetricValues {
+    string text = 1;
```

Comment:
Looked briefly into whether bytes would make more sense, but it shouldn't matter at all AFAICT.